src/uv-common.h

```diff
@@ -61,12 +65,16 @@ extern int snprintf(char*, size_t, const char*, ...);
   void uv__static_assert(int static_assert_failed[1 - 2 * !(expr)])
 #endif
 
-#if defined(__GNUC__) && (__GNUC__ > 4 || __GNUC__ == 4 && __GNUC_MINOR__ >= 7)
-#define uv__load_relaxed(p) __atomic_load_n(p, __ATOMIC_RELAXED)
-#define uv__store_relaxed(p, v) __atomic_store_n(p, v, __ATOMIC_RELAXED)
+#ifdef _MSC_VER
+#define uv__load_int_relaxed(p)                                               \
+  InterlockedExchangeNoFence((LONG volatile*)(p), p)
+#define uv__store_int_relaxed(p, v)                                           \
+  (*((LONGLONG volatile*) p) = v)
```

Comment:
Sorry, I probably didn't communicate that clearly but InterlockedExchangeNoFence() is for _setting_ the value.

Reply:
Yup. That makes more sense.